.github/dependabot.yml

```
@@ -10,6 +10,14 @@ updates:
         update-types: ["version-update:semver-major"]
       - dependency-name: "@types/node"
         update-types: ["version-update:semver-major"]
+    groups:
+      aws-sdk:
+        patterns:
+        - "@aws-sdk/*"
+      eslint:
+        patterns:
+        - "@typescript-eslint/*"
+        - "eslint/*"
```

I don't think `eslint/*` matches anything - there's the base `eslint` package that does not have a trailing slash, and several packages that start with `eslint-`, but I don't see anything that looks like `eslint/` in `npm list`.

Maybe this should just be a single, broad match?

```suggestion
        - "*eslint*"
```

Optionally, we can also limit the scope to `devDependencies`:

```yaml
dependency-type: development
```

([reference](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups))

---

Oh thanks good catch -- I was avoiding a wide match like that just because of concern that it would some day match unrelated dependencies... but just for fun I spent 60 seconds using the npm search to search "eslint{X}" for all the possible values of `{X}` and... based on a very unscientific analysis there do not appear to be any non-eslint-packages that include the string `eslint` in the title.